test/Sign.Core.Test/SignatureProviders/AggregatingSignatureProviderTests.cs

```diff
@@ -147,7 +147,7 @@ public async Task SignAsync_WhenFilesIsNull_Throws()
             ArgumentNullException exception = await Assert.ThrowsAsync<ArgumentNullException>(
                 () => provider.SignAsync(
                     files: null!,
-                    new SignOptions(HashAlgorithmName.SHA256)));
+                    new SignOptions(HashAlgorithmName.SHA256, new Uri("http://timestamp.test"))));
```

Comment:
Can we use the `_options` static variable instead of creating a new `SignOptions` object?